packages/ui/html-tooltip/index.module.css

```
@@ -0,0 +1,27 @@
+  anchor-name: --my-tooltip;
+}
+
+.tooltip {
+  opacity: 0;
+  visibility: hidden;
+  position-anchor: --my-tooltip;
+  position: fixed;
+  position-area: bottom;
+  position-try-fallbacks: top;
+
+  background: #333;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)